src/i18n/locales/fr/common.json

```
@@ -0,0 +1,151 @@
+  "KoboldAI": "KoboldAI",
+  "Window.ai": "Window.ai",
+  "Name": "Nom",
+  "System Prompt": "Invite système",
+  "TTS": "TTS",
+  "TTS Backend": "Backend TTS",
+  "Piper": "Piper",
+  "ElevenLabs": "ElevenLabs",
+  "SpeechT5": "SpeechT5",
+  "Coqui": "Coqui",
+  "OpenAI": "OpenAI",
+  "Vision Backend": "Backend Vision",
```

Biome (2.1.2)

[error] The key SpeechT5 was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)